source/_integrations/switchbot_cloud.markdown

```diff
@@ -0,0 +1,39 @@
+---
+title: SwitchBot Cloud
+description: Instructions on how to set up SwitchBot Devices.
+ha_category:
+  - Switch
+ha_release: '2023.10'
+ha_iot_class: Cloud Polling
+ha_codeowners:
+  - '@SeraphicRav'
+ha_domain: switchbot_cloud
+ha_bluetooth: false
+ha_platforms:
+  - switch
+ha_config_flow: true
+ha_integration_type: integration
+---
+
+The SwitchBot integration allows you to control SwitchBot [devices](https://www.switch-bot.com/).
```

Comment:
```suggestion
The SwitchBot Cloud integration allows you to control SwitchBot [devices](https://www.switch-bot.com/).
```
Perhaps also explain more about switchbot cloud. Since Switchbot now has 2 integrations, one for cloud one for bluetooth, I, as a reader would like to know which I should use. Do I need a hub with it? Does this also support bluetooth devices if they are connected via the hub? (and if that is something for the future you can omit it as you have a supported device list).

Clearing this up in the first few sentences saves you a lot of issues of people thinking if this the switchbot integration they should use (people don't read)